connectors/jersey2/pom.xml

```diff
@@ -28,7 +28,12 @@
         <dependency>
             <groupId>org.glassfish.jersey.media</groupId>
             <artifactId>jersey-media-json-jackson</artifactId>
-            <version>2.11</version>
+            <version>${jersey-version}</version>
+        </dependency>
+        <dependency>
```

Comment:
Without this additional dependency the glassfish applicationserver will not start du to the missing InjectionManager which was extracted at version 2.34 into a separate dependency